linera-views/src/backends/rocks_db.rs

```diff
@@ -64,15 +63,6 @@ pub struct RocksDbStoreConfig {
     pub common_config: CommonStoreConfig,
 }
 
-#[derive(Default)]
-struct RocksDbStores {
-    stores: BTreeMap<(String, Vec<u8>), RocksDbStoreInternal>,
```

Comment:
Were we using `RocksDbStore` at all in the previous version?

Reply:
Oh I see, never mind.

Reply:
Definitely for the memoization of the opened stores.